db/schema.rb

```diff
@@ -10,10 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_01_25_180747) do
+ActiveRecord::Schema.define(version: 2022_02_02_103752) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "address_standardizer"
```

Comment:
This line here got put back in when I ran the migration locally. Maybe shouldn't be removed?